### ISSUE TITLE
What kind-of icons do you want linter to show in gutter?

### ISSUE THREAD

**Issue by steelbrain** [philschatz/atom-pull-requests#19]
You mentioned it in the README, can you provide a screenshot or something explaining the idea

**Comment by philschatz** [comment]
Thanks for taking a look into this! I was hoping to be able to use octicons in the gutter and in the linter messages.

I was able to add the comment icon (see the following screenshot) by overriding the `info` circle that normally shows up in the gutter. Unfortunately, it's an ugly CSS hack and comes with problems:

1. if any other linter runs on the page and uses the `info`, their icon will also be a comment icon
2. to get the icon to show up I had to mangle the CSS which generates a circle by default (see [source](https://github.com/philschatz/atom-pull-requests/blob/master/styles/pull-requests.atom-text-editor.less#L10-L29)
3. If a user uses a custom theme this hack causes problems (I don't remember exactly how/when but someone reported it about a year ago)

![](https://cloud.githubusercontent.com/assets/253202/11326511/82360626-9139-11e5-8466-ed2d356cb0d8.png)

I hope that helps, and if I can test a branch, I'd be happy to!
